src/Migrations/20231127171339_RenamedDisableNode.Designer.cs

```diff
@@ -0,0 +1,1263 @@
+﻿// <auto-generated />
```

Comment:
There should only be 1 designer file, for creating IsNodeEnabled. Please remove all of the designer and migrations you created on this branch, drop the database, and redo the migration